code/controllers/subsystem/minimap.dm

```diff
@@ -500,6 +503,8 @@ SUBSYSTEM_DEF(minimaps)
 		qdel(svg_store_overlay)
 		debug_log("SVG coordinates for [faction] are not implemented!")
 
+#define can_draw(faction, user) (faction == FACTION_MARINE && skillcheck(user, SKILL_LEADERSHIP, SKILL_LEAD_EXPERT) || faction == XENO_HIVE_NORMAL && isqueen(user))
```

Comment:
Can you add parens to explicit order here? a && b || c && d is ambiguous

Reply:
Order of precedence is here: https://www.byond.com/docs/ref/#/operator but sure.